tests/vmi_multiqueue_test.go

```diff
@@ -123,5 +123,35 @@ var _ = Describe("[Serial]MultiQueue", func() {
 			}
 		})
 
+		It("should be able to create a multi-queue VMI when requesting a single vCPU", func() {
+			vmi := tests.NewRandomFedoraVMIWitGuestAgent()
+			multiQueue := true
+
+			// by default, we request a single vCPU
```

Comment:
Can we still make it explicit? Probably bletter, than if we for whatever reason ever change that, and the test would stay green because we test a different scenario than what we expect.

Reply:
+1 And no comments :)

Reply:
Done.